index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Portfolio by Abenezer Samuel</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header class="toolbar">
+    <nav class="nav-items">
+      <h2 class="logo">Ebenezer</h2>
```

Comment:
- Hey, please, kindly move the logo in this case which is your name out of the nav, and make it a link so it can be clickable to redirect users to the homepage.